Task6.08

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+
+def replace(array, number, replacement_number) -> (tuple[int, int], int, int):
+    """
+    NumPy program to replace all numbers in a given array which is equal, less and greater to a given number
+    :param array:
+    :param number:
+    :param replacement_number:
+    :return: nothing
+    """
+    print("array before replacement:\n", array)
+    print(f"\nReplace elements that equals to {number} with {replacement_number}")
+    print(np.where(array == number, replacement_number, array))
+    print(f"\nReplace elements that less than {number} with {replacement_number}")
+    print(np.where(array < number, replacement_number, array))
+    print(f"\nReplace elements that greater than {number} with {replacement_number}")
+    print(np.where(array > number, replacement_number, array))
```

Comment:
You should seperate logic from output, there is nor reason to print withing this function.